source/_cookie_banner.html.erb

```diff
@@ -0,0 +1,21 @@
+<div id="cookie-banner" class="govuk-cookie-banner" hidden="true" data-nosnippet role="region" aria-label="Cookies on <%= config[:service_name] %>" data-module="cookie-banner">
```

Comment:
In the screenshot the heading of the cookie banner is just "Cookies on". It should be "Cookies on GOV.UK Forms". I guess that's something to do with this service_name variable so might be ok in real life?

Reply:
good spot - I'd removed the service_name by mistake so I've added it back in